### PR TITLE
Revamp the binary install guide

### PR DESCRIPTION
**Pull request by mstanleyjones**
### Describe the proposed changes

Rewrites to the binary installation guide to clarify the URL scheme for download links and to
streamline installation instructions

### Related issue

#700 
#704 

### Please take a look

@thaJeztah @andrewhsu @vikstrous @justincormack @friism 

Please use the Netlify preview to view the output.

### CODE REVIEW

**Comment by johndmulhausen** [comment]
Deploy preview ready!

Built with commit 33808341db32566a672ad979adb2892326b6019c

https://deploy-preview-764--docker-docs.netlify.com

**Comment by johndmulhausen** [comment]
Deploy preview ready!

Built with commit dedc4d88e768eaa9a8af235cf95b13f5290caf6d

https://deploy-preview-764--docker-docs.netlify.com

File: engine/installation/binaries.md [hunk 1]
```diff
@@ -81,37 +81,54 @@ exhibit unexpected behaviour.
...
-
-    https://get.docker.com/builds/Linux/i386/docker-<version>.tgz
-
-    https://get.docker.com/builds/Linux/x86_64/docker-<version>.tgz
-
-For example:
+You can only download the Windows binary for version `1.9.1` onwards.
+Moreover, the 32-bit (`i386`) binary is only a client, you cannot use it to
+run the `docker` daemon. The 64-bit binary (`x86_64`) is both a client and
+daemon.
```
Comment (friism) [inline engine/installation/binaries.md]: I think it's more accurate to say something like "The "i386" download contains a 32bit client-only binary. The "x86_64" download contains both client and daemon binaries for 64bit Windows Server 2016 and Windows 10 systems."

File: engine/installation/binaries.md [hunk 2]
```diff
@@ -81,37 +81,54 @@ exhibit unexpected behaviour.
...
-following URL patterns:
-
-    https://get.docker.com/builds/Linux/i386/docker-<version>.tgz
-
-    https://get.docker.com/builds/Linux/x86_64/docker-<version>.tgz
-
-For example:
+You can only download the Windows binary for version `1.9.1` onwards.
+Moreover, the 32-bit (`i386`) binary is only a client, you cannot use it to
+run the `docker` daemon. The 64-bit binary (`x86_64`) is both a client and
```
Comment (friism) [inline engine/installation/binaries.md]: `docker` -> `dockerd`

File: engine/installation/binaries.md [hunk 3]
```diff
@@ -81,37 +81,54 @@ exhibit unexpected behaviour.
...
+daemon.
 
-    https://get.docker.com/builds/Linux/i386/docker-1.11.0.tgz
+The macOS binary is only a client. You cannot use it to run the `docker`
```
Comment (friism) [inline engine/installation/binaries.md]: `docker` -> `dockerd`

File: engine/installation/binaries.md [hunk 4]
```diff
@@ -81,37 +81,54 @@ exhibit unexpected behaviour.
...
 
-    https://get.docker.com/builds/Linux/x86_64/docker-1.11.0.tgz
+### URL patterns for static binaries
+
+The URLs for released binaries are stable and follow a predictable pattern.
+Unfortunately, it is not possible to browse the releases in a directory structure.
+If you do not want to get the links from the release notes for a release, you can
+infer the URL for the binaries by using the following patterns:
+
+| Description            | URL pattern                                                       |
```
Comment (friism) [inline engine/installation/binaries.md]: @dmp42 @andrewhsu can we start using `download.docker.com` instead of metastasizing these ones?

**Comment by vikstrous** [comment]
I just noticed this, but it's unrelated: https://github.com/docker/docker.github.io/issues/784
The changes look great!

**Pull request merged**